R/create_target_design.R

```diff
@@ -109,18 +126,18 @@ create_target_design <- function(
     stop("Either targetmin + targetmax or midpoint must be supplied")
   }
   if (!is.null(targetmin) && !is.null(targetmax)) {
-    if (!is_single_valid_number(targetmin) || !is_single_valid_number(targetmax)) {
-      stop("targetmin or targetmax misspecified/not numeric")
+    if (!is.numeric(targetmin) || !is.numeric(targetmax) || length(targetmin) != length(targetmax)) {
```

Comment:
throughout this function you've replace `is_single_valid_number` with `is.numeric`. since we now support vectors of length > 1, this is a necessary change. however, `is_single_valid_number` also checks that the target is non-NA and finite (not Inf, not NaN). Should we add a new helper function for is_valid_number or something?